CHANGELOG.next.toml

```diff
@@ -35,3 +35,12 @@ message = "Fix compiler errors in generated code when naming shapes after types
 references = ["smithy-rs#2696"]
 meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client"}
 author = "jdisanti"
+
+[[smithy-rs]]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
+message = """`ShapeId` is the new structure used to represent a shape, with its absolute name, namespace and name.
+`OperationExtension`'s members are replaced by the `ShapeId` and operations' names are now replced by a `ShapeId`.
```

Comment:
I think, as this is a breaking change we should have a tiny migration guide here. This should demonstrate the difference in the API in the context of `filter_by_shape_name` vs `filter_by_shape_id`, and `OperationExtension`.